pyquil/operator_estimation.py

```diff
@@ -750,7 +750,9 @@ def serializable(self):
 
 def measure_observables(qc: QuantumComputer, tomo_experiment: TomographyExperiment,
                         n_shots: int = 1000, progress_callback=None, active_reset=False,
-                        readout_symmetrize: str = None, calibrate_readout: str = None):
+                        readout_symmetrize: str = None, calibrate_readout: str = None,
```

Comment:
I think this is unnecessarily complicated.  If inheriting these error model properties are necessary to get the expect behavior out of the fox, they should all default to `True`. Moreover, not inheriting them will lead to unexpected behavior (calibration misbehaving) for some problems, so why allow it at all? Until the way we fix the way we specific error model (i.e., not via programs), then `measure_observables` needs to inherit everything for correctness.

Reply:
Setting the default values of the other inheritance choices to `True` would result in unexpected behavior, e.g. in DFE (as far as I understand it). I don't see how `test_process_dfe_bit_flip` would pass for example with the other choices set to `True`.

Reply:
As discussed yesterday, if the bitflip is faulty, all usage of the bitflip should be faulty, including calibration -- calibration needs to be representative of what happens during the computation that is being characterized.